src/pages/projects/AddProjectManagerForm.tsx

```diff
@@ -77,6 +83,21 @@ export const AddProjectManagerForm = ({
     }
   };
 
+  const addSelfAsProjectManager = (person: CristinPerson) => {
+    if (person.affiliations.length > 1) {
+      setSearchTerm(getFullCristinName(person.names));
+    } else {
+      addProjectManager(person);
+      toggleModal();
+    }
+  };
+
+  const currentUserQuery = useRefetchPerson({
+    userCristinId: userCristinId,
```

Comment:
Burde kanskje være tydligere på skille mellom user og person her, da dette egentlig ikke har noe med NVA-User å gjøre?
```suggestion
  const currentPersonQuery = useRefetchPerson({
    personId: userCristinId, // ev. cristinId elns
```